### DISCUSSION TITLE
How does pandoc decide whether something is a block quote in Word and LibreOffice documents?

### DISCUSSION THREAD

**Discussion by kopeckyf** [jgm/pandoc#9382] [Q&A]
I've read that pandoc identifies whether a paragraph is a block quote by looking at indentation. But how much indentation, exactly, is required by pandoc to trigger a block quote? 

In LibreOffice, I noticed that setting even an outrageously large indent does not trigger pandoc's block quote detection. But hitting “increase indent” from the toolbar does the trick:
![grafik](https://github.com/jgm/pandoc/assets/10461729/de73e35b-f9f8-48a4-a23a-12fbcdb4e5c3)

Is there a general rule about how pandoc will determine whether something is a block quote?

**Comment by jgm** [comment]
In docx, the rule is:

- paragraph is indented relative to its context, and not a list paragraph
- or, style is one of :     "Quote", "Block Text", "Block Quote", "Block Quotation", "Intense Quote"

For ODT the relevant code is here: https://github.com/jgm/pandoc/blob/djot/src/Text/Pandoc/Readers/ODT/ContentReader.hs#L394-L431
